modules/log/logger_global.go

```diff
@@ -34,6 +34,10 @@ func Debug(format string, v ...any) {
 	Log(1, DEBUG, format, v...)
 }
 
+func DebugWithSkip(skip int, format string, v ...any) {
+	Log(skip+1, DEBUG, format, v...)
+}
+
```

Comment:
You don't need it

Reply:
acf8026a4b777d48b1089bce16b71b62c2e83e32